tests/config/nextflow.config

```diff
@@ -25,6 +25,10 @@ if ("$PROFILE" == "singularity") {
     podman.enabled = true
     podman.userEmulation = true
     podman.runOptions = "--runtime crun --platform linux/x86_64 --systemd=always"
+} else if ("$PROFILE" == "gpu") {
```

Comment:
Does `if else` mean that only one of the profiles will be active at a time? For a successful test with docker/gpu, we need both docker and gpu?

nf-test uses the nf-test.config from the project root directory (https://github.com/nf-core/modules/blob/473857e48869a078b64932241acb1cc742460530/nf-test.config) to locate the profiles defined in the [tests/config/nf-test.config](https://github.com/nf-core/modules/blob/473857e48869a078b64932241acb1cc742460530/tests/config/nf-test.config). Should we also define the gpu profile there?

Reply:
added the missing profile